anta/tests/interfaces.py

```diff
@@ -346,33 +346,66 @@ class VerifyL3MTU(AntaTest):
     NOT_SUPPORTED_INTERFACES: List[str] = ["Management", "Loopback", "Vxlan", "Tunnel"]
 
     @AntaTest.anta_test
-    def test(self, mtu: int = 1500) -> None:
+    def test(self, mtu: int = 1500, ignored_interfaces: Optional[List[str]] = None, specific_mtu: Optional[List[Dict[str, int]]] = None) -> None:
         """
-        Run VerifyL3MTU validation
+        Verifies the global L3 Maximum Transfer Unit (MTU) for interfaces.
 
-        Args:
-          mtu: Layer 3 MTU to verify. Defaults to 1500.
+        Test that interfaces are configured with the correct MTU. It supports Ethernet, Port Channel and VLAN interfaces.
+        You can define a global MTU to check and also an MTU per interface and also ignored some interfaces.
```

Comment:
I think we should move this to the class docstring instead of the Limitations section and remove the TOI link. We should also be clear that by default the Management, Loopback, Vxlan, Tunnel interfaces are not being checked by the test.